libp2p/P2PMessage.h

```diff
@@ -106,5 +106,22 @@ enum AMOPPacketType
     RequestTopics = 2,
     SendTopics = 3
 };
+enum TopicStatus
+{
+    INIT_STATUS = 0,
+    ENABLE_STATUS = 1,
```

Comment:
can this `ENABLE_STATUS` more meaningful?

Reply:
VERIFYING_STATUS = 0,                     
    VERIFYI_SUCCESS_STATUS = 1,                           
    VERIFYI_FAILED_STATUS = 2,  
change to this